scripts/package/tools/build

```diff
@@ -0,0 +1,100 @@
+#! /usr/bin/env bash
+
+# Builds a specific package in the context of FPM's Docker container.
+# https://fpm.readthedocs.io/
+
+# Enforce preconditions
+set -ex
+version="$1"
+install_dir=/usr/lib/pash
+set +x -u; : "$version"
+shift # for remaining $@
+
+
+print_dotfpm() {
+    output_format="$1"
+    output_file="$2"
+    
+    # I/O rules, with Docker container paths
+    flag --chdir /src
+    flag --input-type dir
+    flag --output-type "$output_format"
+    flag --package "$output_file"
+    flag --log info
+    flag --verbose
+    flag --force
+
+    # Discovery information
+    flag --name pash
+    flag --description "Parallelize POSIX shell scripts"
+    flag --license mit
+    flag --architecture all
+    flag --url 'https://github.com/binpash/pash'
+    flag --version "$version"
+
+
+    # System-level dependencies
+    flag --depends automake
+    flag --depends bash
+    flag --depends bc
+    flag --depends bsdmainutils
+    flag --depends curl
+    flag --depends git
+    flag --depends g++-10
```

Comment:
I remember `g++-10` dependency needing special attention with a new apt repository for the sake of some special optimizations. Is that a must-have for the first working version of this branch?

Reply:
Dropping this, since no one is responding. I put a function for the `g++-10` alternative in `utils.sh`